guides/gatherers.md

```diff
@@ -163,48 +220,60 @@ This gatherer allows accessing the hostnames that are resolvable through `/etc/h
 does **not** use domain resolution in any way but instead directly parses the file.
 
 It allows one argument to be specified or none at all:
- - When a hostname is provided as an argument, the gatherer will return an array of IPv4 and/or IPv6 addresses.
- - When no argument is provided, the gatherer will return a map with hostname as keys and arrays with IPv4 and/or IPv6 addresses.
 
-Sample arguments:
+- When a hostname is provided as an argument, the gatherer will return an array of IPv4 and/or IPv6 addresses.
+- When no argument is provided, the gatherer will return a map with hostname as keys and arrays with IPv4 and/or IPv6 addresses.
 
-| Name                                | Return value                                                                                            |
-|:------------------------------------|:--------------------------------------------------------------------------------------------------------|
-|`localhost`                          | list of IPs resolving `localhost` e.g. `["127.0.0.1", "::1"]`                                           |
-|`node1`                              | list of IPs resolving `node1` e.g. `["172.22.0.1"]`                                                     |
-|`no argument provided`               | map with hostnames and IP addresses e.g. `{"localhost": ["127.0.0.1", "::1"], "node1": ["172.22.0.1"]}` |
+Example arguments:
+
+| Name                   | Return value                                                                                            |
+| :--------------------- | :------------------------------------------------------------------------------------------------------ |
+| `localhost`            | list of IPs resolving `localhost` e.g. `["127.0.0.1", "::1"]`                                           |
+| `node1`                | list of IPs resolving `node1` e.g. `["172.22.0.1"]`                                                     |
+| `no argument provided` | map with hostnames and IP addresses e.g. `{"localhost": ["127.0.0.1", "::1"], "node1": ["172.22.0.1"]}` |
+
+Example specification:
 
-Specification examples:
 ```yaml
 facts:
   - name: hosts_node1
     gatherer: hosts
     argument: node1
-  
+
   - name: hosts_node2
     gatherer: hosts
     argument: node2
 
   - name: hosts_all
     gatherer: hosts
-
 ```
 
-For more information refer to [trento-project/agent/../gatherers/hostsfile_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/hostsfile_test.go)
+Example output (in Rhai):
+
+```ts
+// hosts_node1
+30000;
+
+// hosts_node2
+30000;
+
+// hosts_all
+"udpu";
+```
```

Comment:
The provided example does not match with the related description.
This gatherer returns either a list of IPs for a provided hostname or a map of all the found entries in hosts file end their relative addresses.